code/__DEFINES/dcs/signals.dm

```diff
@@ -173,6 +173,8 @@
 #define COMSIG_MOVABLE_UNBUCKLE "unbuckle"						//from base of atom/movable/unbuckle_mob(): (mob, force)
 #define COMSIG_MOVABLE_PRE_THROW "movable_pre_throw"			//from base of atom/movable/throw_at(): (list/args)
 	#define COMPONENT_CANCEL_THROW (1<<0)
+#define COMSIG_MOVABLE_LIVING_THROW_IMPACT_CHECK "movable_living_throw_impact_check" //from base of /atom/movable/proc/hit_check
```

Comment:
This comment is still inaccurate for the same reason

Reply:
Also signals should not be named CHECK or SET, this is a LIVING_PRE_IMPACT signal by signal naming standards